packages/flutter_goldens/lib/skia_client.dart

```diff
@@ -579,7 +579,11 @@ class SkiaGoldClient {
     final Map<String, Object?> parameters = <String, Object?>{
       if (_isBrowserTest)
         'Browser' : _browserKey,
-      'Abi': '$abi',
+      // This method is only used in local testing to look up the given image.
+      // CI is not comprehensive in possible abi keys, so false negatives can
+      // be produced if we include it in local testing, for example CI uses
```

Comment:
Small comment nits: `testing, for` -> `testing. For` and `but a contributor could have macos_arm64` -> `but a contributor could have macos_arm64.`